### PR TITLE
Newfeature/issue 7 create footer component

### PR DESCRIPTION
**Pull request by Peter-holyoke**
## Related Issue

https://github.com/ProvidenceGeeks/website-frontend/issues/7

## Summary of Changes

- [x] Completed footer component

~***Note:*** The `<hr>` in the footer is not perfectly responsive, open to suggestions on this.~
Fixed using flexbox/bootstrap.

### CODE REVIEW

File: src/components/footer/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+      <div className="footer row">
+        <div className="footer-top d-flex">
+          <div className="col-md-2">
+            <div className="logo">
+              <a href="/"><PvdGeeksLogo /></a>
+            </div>
+          </div>
+
+          <div className="social-container col-md-8 offset-md-2 d-flex justify-content-end align-items-center">
```
Comment (thescientist13) [inline src/components/footer/footer.jsx]: I wonder if this makes the case for a reusable component for these social network links?  Something _social-networking-links_

**Comment by thescientist13** [comment]
@Peter-holyoke 
> Note: The <hr> in the footer is not perfectly responsive, open to suggestions on this.

Could you elaborate?  In what way is it not, and in what way would you like it to be?

**Comment by Peter-holyoke** [comment]
Sure. On smaller resolutions and the mobile version of the site, the `<hr>` is not the appropriate width, because I used a percentage for the CSS. It should scale correctly on all resolutions, but I had some issues getting that to work perfectly.

**Comment by thescientist13** [comment]
@Peter-holyoke 
Sorry, meant to say _develop_ [here](https://github.com/ProvidenceGeeks/website-frontend/pull/32#pullrequestreview-72243564)

File: src/components/footer/footer.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const twitterLink = footer.find('.social-container .twitter-link');
+
+    expect(twitterLink.prop('href')).toEqual('https://twitter.com/providencegeeks');
+    expect(twitterLink.prop('title')).toEqual('Follow Us');
+    expect(twitterLink.prop('target')).toEqual('_blank');
+    expect(twitterLink.length).toEqual(1);
+    expect(twitterLink.find('svg.twitter-icon').length).toEqual(1);
+  });
+
+  it('should have copyright notice with current year', () => {
```
Comment (thescientist13) [inline src/components/footer/footer.spec.jsx]: 🎉

**Pull request merged**